### DISCUSSION TITLE
Extend the move-images-to-other-tab.ini Command

### DISCUSSION THREAD

**Discussion by moritzdietz** [hluk/CopyQ#1536] [Q&A] [answered]
I just discovered this "Discussion" Tab in GitHub and am using it for the first time now - let's see how this goes and if someone even sees this.

I am trying to extend the [move-images-to-other-tab.ini](https://github.com/hluk/copyq-commands/blob/master/Automatic/move-images-to-other-tab.ini) Command so that it can handle multiple MIME Types additionally to `image/png` but I can't make it work. I thought of using `image/*` but this just resulted in nothing of type `image/` matching it.
Any ideas?

**Comment by hluk** [comment, marked as answer]
The Format/Input field doesn't support patterns, but you can use this command instead:

```ini
[Command]
Automatic=true
Command="
    copyq:
    var imageTab = '&Images'
    var formats = dataFormats()
    for (var i in formats) {
        if (formats[i].match(/^image\\//)) {
            setData(mimeOutputTab, imageTab)
            break
        }
    }"
Icon=\xf03e
Name=Move Images to Other Tab
```

**Reply by moritzdietz** [reply]
Aha! That's very good to know and explains the behaviour. Thanks for writing the command for me - this is very much appreciated.
Should I open up a PR to replace this command with the command I mentioned above? Possibly this could be also of interest to others.

**Reply by hluk** [reply]
Glad I could help. Feel free to open the PR.
